src/templates/ja/2019/methodology.html

```diff
@@ -0,0 +1,433 @@
+            <td>
+              Redwood City, California, USA<br>
+              The Dalles, Oregon, USA
+            </td>
+          </tr>
+
+          <tr>
+            <td>接続方法</td>
+            <td>ケーブル(5/1 Mbps 28ms RTT)</td>
+            <td>3G (1.600/0.768 Mbps 300ms RTT)</td>
+          </tr>
+
+          <tr>
+            <td>ビューポート</td>
+            <td>1200 x 1920px</td>
+            <td>512 x 360px</td>
+          </tr>
+        </tbody>
+      </table>
+      </div></div>
+
+      <p>
+        デスクトップのWebサイトは、Linux VM上のデスクトップChrome環境内から実行されます。ネットワーク速度はケーブル接続と同等です。
+      </p>
+
+      <p>
+        モバイルWebサイトは、3G接続と同等のネットワーク速度を持つエミュレートされたMoto G4デバイス上のモバイルChrome環境から実行されます。エミュレートされたモバイルユーザーエージェントはChrome65と自称していますが、実際は Chrome75であることに注意してください。
+      </p>
+
+      <p>
+        テストが実行される場所は2つあります。カリフォルニア州とオレゴン州です。HTTP Archiveは、カリフォルニア州にある<a href="https://archive.org">Internet Archive</a>のデータセンターにある独自のテストエージェントハードウェアを維持しています。オレゴン州にある<a href="https://cloud.google.com/compute/docs/regions-zones/#locations">Google Cloud Platform</a> のus-west-1にあるテストエージェントは、必要に応じて追加されています。
+      </p>
+
+      <p>
+        HTTP ArchiveのWebPageTestのプライベートインスタンスは、最新のパブリックバージョンと同期して維持され、<a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">カスタムメトリクス</a>で強化されています。これらは、テストの最後に各ウェブサイトで評価されるJavaScriptのスニペットです。<a href="https://github.com/HTTPArchive/legacy.httparchive.org/blob/master/custom_metrics/almanac.js">almanac.js</a>カスタムメトリクスには、DOMの状態に依存するメトリクスなど、そうでなければ計算できないいくつかのメトリクスが含まれています。
+      </p>
+
+      <p>
+        各テストの結果は、<a href="https://en.wikipedia.org/wiki/HAR_(file_format)">HARファイル</a>として公開され、ウェブページに関するメタデータを含むJSON形式のアーカイブファイルです。
+      </p>
+    </section>
+
+    <section>
+      <h3 id="lighthouse"><a href="#lighthouse" class="anchor-link">Lighthouse</a></h3>
+
+      <p>
+        <a href="https://developers.google.com/web/tools/lighthouse/">Lighthouse</a>は、Googleが構築した自動化されたウェブサイト品質保証ツールです。最適化されていない画像やアクセスできないコンテンツなどのユーザー体験のアンチパターンが含まれていないことを確認するためにウェブページを監査します。
+      </p>
+
+      <p>
+        HTTP Archiveは、すべてのモバイルWebページでLighthouseの最新バージョンを実行しています - リソースが限られているため、デスクトップページは含まれていません。2019年7月のクロール時点で、HTTP Archiveは<a href="https://github.com/GoogleChrome/lighthouse/releases/tag/v5.1.0">5.1.0</a>バージョンのLighthouseを使用していました。
+      </p>
+
+      <p>
+        Lighthouseは<a href="#webpagetest">WebPageTest</a>の中から独自のテストとして実行されますが、独自の設定プロファイルを持っています。
+      </p>
+      <div class="table-wrap"><div class="table-wrap-container">
+      <table>
+        <thead>
+          <tr>
+            <th>設定</th>
+            <th>値</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>CPU slowdown</td>
+            <td>1x*</td>
+          </tr>
+          <tr>
+            <td>Download throughput</td>
+            <td>1.475 Mbps</td>
+          </tr>
+          <tr>
+            <td>Upload throughput</td>
+            <td>0.675 Mbps</td>
+          </tr>
+          <tr>
+            <td>RTT</td>
```

Comment:
Should we translate all these config titles (or at least the first 3)?

Reply:
At first, I don't need translate.
But Now, I thought need translate at first 3.
I translated.